### ISSUE TITLE
should ribs be classified as long bones?

### ISSUE THREAD

**Issue by dosumis** [obophenotype/uberon#2010]
**Uberon term**

rib http://purl.obolibrary.org/obo/UBERON_0002228
![image](https://user-images.githubusercontent.com/112839/127642892-b9b25324-be79-4792-b4fb-2e17a528ed70.png)

**Bug description**

The typology here: https://en.wikipedia.org/wiki/Long_bone
and the definition here: https://medlineplus.gov/ency/imagepages/9582.htm 
are not consistent with rib being a long bone.  Should instead be classified as a flat bone.

I guess some of these classifications might not be stable across vertebrate evolution, but it is important we can capture standard mamalian/human classifications.

**Comment by dosumis** [comment]
CC @RDruzinsky @wdahdul  - would be good to get your input here.

**Comment by wdahdul** [comment]
I'm not as familiar with the mammalian terminology, but agree ribs shouldn't be classified as long bones. Not sure about classifying them as flat bones, but @RDruzinsky can give the definitive answer

**Comment by RDruzinsky** [comment]
A rib is a long bone in the sense that it is an endochondral bone with a
diaphysis and an epiphysis. Traditionally these types of bones have been
grouped together as long bones. In contrast, the bones of the skull vault
that develop from mesenchyme are the ones that are usually called flat
bones. All bones do not fit easily into this classification scheme. The
scapula is, more or less, flat, but we don't call it a flat bone because
it is an endochondral bone.
Although, in normal usage, it is not incorrect to call a rib a long bone, I
think that is confusing and I see no reason for that to be part of the
classification in Uberon, unless someone has a particular reason for
classifying bones by shape.

Robert E. Druzinsky, Ph.D.
Clinical Associate Professor
Dept. of Oral Biology
College of Dentistry
University of Illinois at Chicago
801 S. Paulina
Chicago, IL 60612
***@***.***

Office: 312-996-0406
Lab: 312-996-0629
Website: www.peerj.com/RobertDruzinsky

**Comment by cmungall** [comment]
FWIW, FMA classifies it as a long bone

EMAPA,MA,NCIT do not

no long bone in ZFA, MESH, and others

note to self: compare classification across species

```bash
pq-ontobee -e -m UBERON=$OBO/UBERON_ "compare_via_xref('UBERON:0002228',_,_,_,_,_)"
```

**Comment by rays22** [comment]
1. I think the current UBERON definition of `long bone UBERON:0002495` is too strict to include the
`clavicle bone UBERON:0001105`
and the
`rib UBERON:0002228`.
, because it says `Long bones are present only in the limbs`.
That restriction would be fine for the `limb long bone UBERON:0003606` sub-class.
The MedlinePlus [definition of long bones](https://medlineplus.gov/ency/imagepages/9582.htm) would be also a good fit for `limb long bone UBERON:0003606`.
We would need a broader definition for long bones to include the ribs.
2. A `long bone UBERON:0002495` is *part of some* `appendicular skeleton UBERON:0002091`.
However, the illustration in the wikipedia reference that is part of the definition of `appendicular skeleton UBERON:0002091` excludes the ribs as being part of the [Appendicular skeleton.](https://en.wikipedia.org/wiki/Appendicular_skeleton)
In other words, the ribs do not seem to be considered part of the appendicular skeleton. This suggests to me that ribs should not be classified as long bones in UBERON, because long bones are defined as being part of the appendicular skeleton.

**Comment by rays22** [comment]
My planned actions:
1. Drop classification of `rib` under `long bone`. This is because there is a consensus in the field that ribs are not part of the `appendicular skeleton`.
2. Remove the restriction that long bones are only in the limbs from the `long bone` free text definition to make it consistent with the existing logical classification.

**Comment by dosumis** [comment]
@rays22  - please check that this inference is no longer present in MP:
MP:0004671 long ribs subClassOf MP:0004695 increased length of long bones

**Comment by sbello** [comment]
@dosumis I just ran the ELK reasoner on the file and no longer see that inference so I think we are good (working on the MP today so I thought I would check)
